### PR TITLE
Zach dev

### PR DESCRIPTION
**Pull request by z-yeti**
Pull all this in since it has the new workflow scripts?

### CODE REVIEW

File: src/sections/slideshow.liquid [hunk 1]
```diff
@@ -53,17 +53,34 @@
...
-    {% endif %}
+    autoplay: {{ section.settings.slide_autoplay_check }},
+    autoplaySpeed: {{ section.settings.slide_speed }} * 1000
+  });
+  console.log('slick was borned');
+}
+
+function killSlick(){
+  $('slider').slick('unslick');
+  console.log('i killed slick');
```
Comment (sturdynut) [inline src/sections/slideshow.liquid]: lol

File: src/styles/sections/footer/newsletter.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  .input-group{
+    display: flex;
+    & .site-footer__newsletter-input{
+      width: 100%;
+      height: 3em;
+      background-color: $color-top-bar-bg;
+      color: $color-nav-links;
+      margin-left: auto;
+      padding-left: 10px;
+      &::-webkit-input-placeholder {
```
Comment (sturdynut) [inline src/styles/sections/footer/newsletter.scss]: Do we need to do all these browser prefixes?

File: README.md [hunk 3]
```diff
@@ -25,9 +25,13 @@ env_name:
...
 Change `env_name` to whatever you want really and then change `password`, `theme_id`, and `store` to your Shopify credentials. (See slate docs on how to obtain these).
 
-Start the site:
+Edit the `watch` script in the `package.json` to your slate env.
+  * `"watch": "concurrently \"slate watch -e YOUR-ENV-NAME-HERE\" \"gulp watch-sections\""`
 
-* `slate watch -e env_name`
+### Start the site:
+
+* `yarn` or `npm run watch`
```
[the remark this answers is not in the document]
Comment (z-yeti) [reply README.md]: yup

File: src/scripts/vendor/stickUp.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ function($) {
 		var stickyMarginB = 0;
 		var currentMarginT = 0;
 		var topMargin = 0;
+		var vartop = 0;
```
[the remark this answers is not in the document]
Comment (z-yeti) [reply src/scripts/vendor/stickUp.js]: no, no, es not. yet at least.

File: src/styles/sections/footer/newsletter.scss (hunk 2 above)
Comment (z-yeti) [reply src/styles/sections/footer/newsletter.scss]: maybe, i dunno

**Pull request merged**